stage1/Makefile

```diff
@@ -6,11 +6,7 @@ OBJCOPY = objcopy
 CFLAGS = -DSMP -isystem ../freebsd-headers/include -Wl,--build-id=none -Os -fno-stack-protector
 LDFLAGS = -T linker.ld -nostartfiles -nostdlib
 
-ifneq ($(filter $(FW), 900 1100),)
```

Comment:
Don't remove.

Reply:
Alright!

Reply:
What's the point in having both this and `#else #error` in the header files?

Reply:
In fact the primary reason why I thought it was a good idea to drop this check is to avoid merge conflicts...